bamboo-specs/bamboo.yaml

```diff
@@ -276,13 +292,83 @@ Release ORCA Code:
+    image: maven.earthdata.nasa.gov/cumulus_orca
+    volumes:
+      ${bamboo.working.directory}: ${bamboo.working.directory}
+      ${bamboo.tmp.directory}: ${bamboo.tmp.directory}
+  tasks:
+  - checkout:
+      force-clean-build: 'true'
+      description: Checkout Default Repository
+  - script:
+      interpreter: SHELL
+      scripts:
+      - |-
+        #!/bin/bash
+        bash bin/deployment-rds.sh
+#job for deploying cumulus and orca
+Deploy cumulus and orca:
+  key: DCO
+  other:
+    clean-working-dir: true
+    # Some plugin configurations are not supported by YAML Specs
+  docker:
+    image: maven.earthdata.nasa.gov/cumulus_orca
+    volumes:
+      ${bamboo.working.directory}: ${bamboo.working.directory}
+      ${bamboo.tmp.directory}: ${bamboo.tmp.directory}
+  tasks:
+  - checkout:
+      force-clean-build: 'true'
+      description: Checkout Default Repository
+
+  - script:
+      interpreter: SHELL
+      scripts:
+      - |-
+        #!/bin/bash
+        bash bin/deployment-cumulus-orca.sh
 variables:
-  ORCA_VERSION: 4.0.0
+  DEPLOYMENT: sandbox
+  ORCA_VERSION: 4.0.3
   RELEASE_FLAG: 'false'
   REPORT_BUILD_STATUS: 'true'
   SECRET_GITHUB_EMAIL: BAMSCRT@0@0@HYUkicwfEqt17JZaZuGYmtmRHrnDySiT+kQkBQa7/Rg7IQgl1lUvbncMUu0Wx2k1ayHojNkh2TdOlF0kJ5tw6Q== #encrypted
   SECRET_GITHUB_TOKEN: BAMSCRT@0@0@nPxyXQXMyE6zm3B1oI8Goma78s0H1fJD4vHy6+dpBQJpjDI2ekbcvCAOI8AyjYiILPhqi2w4fT0fx9MFVXxz2JPTp5uPNzG3GM/8XSUgvYo= #encrypted
   SECRET_GITHUB_USER: BAMSCRT@0@0@fFtitNITDsAjJqK2ZXjhnBZ96z2775QLzGR1Xz557gSl8HxxBaTD3Fbmp+HUbpfx  #encrypted
+  AWS_DEFAULT_REGION: us-west-2
+  AWS_ACCESS_KEY_ID: BAMSCRT@0@0@kGZXFxP372YNHoX+IOMW5aJ0UqNRcVn23p3pdd1N2cE=
+  AWS_SECRET_ACCESS_KEY: BAMSCRT@0@0@L81dje5pniBmplVLwbMq7rFxJVHTSMyK1+fDzKitigInHekJyDfRHMJ3W/o+0WWn
+  RDS_SECURITY_GROUP: sg-0c325ffd7caa65675
+  RDS_ENGINE_VERSION: 10.18
+  RDS_USER_ACCESS_SECRET_ARN: BAMSCRT@0@0@oyGx+9yliRZRK603kejPFjG05yRMm4k1OWkLOz8GGN2+VhBV4FbYWz2jXlx+a35L0JHB7jahUDFmvZBp49YZ8MYQbaDZaduFri0JJTe53U7+22P/EO7T6Q3MMj7sBlBOidQoo+mm/qbnRM3fTwKHhw==
+  AWS_SUBNET_ID1: subnet-0b7acca747c16c895
+  AWS_SUBNET_ID2: subnet-0d3416cbfaaf3508e
+  VPC_ID: vpc-0c4932fcc4b86cff6
+  AWS_ACCOUNT_ID: AMSCRT@0@0@AuBwYN809zPGzk1MH5DRPQ==
+  ROLE_BOUNDARY: NGAPShRoleBoundary
+  ORCA_RELEASE_BRANCH: release/v11.1.1-v4.0.1
+  ORCA_TEST_BRANCH: feature/ORCA-test-bamboo
+  PREFIX: rhassanorca
+  DB_ADMIN_USERNAME: postgres
+  DB_ADMIN_PASSWORD: BAMSCRT@0@0@AuBwYN809zPGzk1MH5DRPQ==
+  DB_USER_PASSWORD: BAMSCRT@0@0@AuBwYN809zPGzk1MH5DRPQ==
+  RDS_CLUSTER_ID: rhorca-cumulus-rds-serverless-default-cluster
+  CMR_PASSWORD: BAMSCRT@0@0@OUkgyKDboRA/V/4Qhgpz/A==
+  CMR_USERNAME: username
+  EARTHDATA_CLIENT_ID: BAMSCRT@0@0@0kgw7HCINxNxv1DrtMHK6A==
+  EARTHDATA_CLIENT_PASSWORD: BAMSCRT@0@0@0kgw7HCINxNxv1DrtMHK6A==
+  DLQ_SUBSCRIPTION_EMAIL: rhassan@contractor.usgs.gov
+  DB_HOST_ENDPOINT: BAMSCRT@0@0@0kgw7HCINxNxv1DrtMHK6A==
+  S3_ACCESS_KEY: dummy
+  S3_SECRET_KEY: BAMSCRT@0@0@0kgw7HCINxNxv1DrtMHK6A==
```

Comment:
I don't understand many of these defaults. Seems like they're holdovers from testing, but don't apply generally.

Reply:
some of them are used for testing but don't apply generally like you said. 

I can do cleanup or leave it there for now.

Reply:
yup already removed other unnecessary variables not relevant to this release.